# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/websockets/basic-auth.any.js.ini
# Commit: 85883a720dab
# Full Hash: 85883a720dab61e82bcdb7391236dc06427466ee
# Author: moz-wptsync-bot <wptsync@mozilla.com>
# Date: 2021-04-07 15:47:02
# Regressor Bug: 1702830
# File Overlap Count: 1
# Description:
#   Bug 1702830 - [wpt-sync] Update web-platform-tests to 336ec39f06d15863a4af6b2aa6592530b37a5aee, a=testonly
#   
#   MANUAL PUSH: wpt sync bot
#   
#   wpt-head: 336ec39f06d15863a4af6b2aa6592530b37a5aee
# ==============================================================================

diff -r acef12d74007 -r 85883a720dab testing/web-platform/meta/websockets/basic-auth.any.js.ini
--- a/testing/web-platform/meta/websockets/basic-auth.any.js.ini	Fri Apr 02 22:03:46 2021 +0000
+++ b/testing/web-platform/meta/websockets/basic-auth.any.js.ini	Fri Apr 02 22:06:52 2021 +0000
@@ -8,13 +8,15 @@
 
 [basic-auth.any.html?wpt_flags=h2]
   expected:
-    if not debug and (os == "win") and (processor == "x86_64"): OK
-    if not debug and (os == "win") and (processor == "x86"): ["TIMEOUT", "OK"]
-    if not debug and (os == "linux") and webrender: TIMEOUT
-    if not debug and (os == "linux") and not webrender: ["TIMEOUT", "OK"]
-    if debug and (os == "linux"): TIMEOUT
-    if not debug and (os == "android"): OK
-    CRASH
+    if debug and (os == "linux") and webrender and not fission and not swgl: ["CRASH", "ERROR", "TIMEOUT"]
+    if debug and (os == "linux") and webrender and not fission and swgl: CRASH
+    if not debug and not webrender and (os == "win") and (processor == "x86"): ["OK", "TIMEOUT"]
+    if debug and (os == "linux") and webrender and fission: ["CRASH", "OK", "TIMEOUT"]
+    if not debug and not webrender and (os == "linux"): ["OK", "TIMEOUT"]
+    if debug and (os == "linux") and not webrender: ["CRASH", "OK", "TIMEOUT"]
+    if debug and (os == "mac"): CRASH
+    if debug and (os == "win"): CRASH
+    if debug and (os == "android"): CRASH
   [HTTP basic authentication should work with WebSockets]
     expected:
       if os == "linux": ["PASS", "FAIL"]
@@ -22,13 +24,16 @@
 
 [basic-auth.any.sharedworker.html?wpt_flags=h2]
   expected:
-    if not debug and (os == "win") and (processor == "x86_64"): OK
+    if debug and (os == "linux") and not swgl and not fission and webrender: ["CRASH", "ERROR", "TIMEOUT", "OK"]
+    if debug and (os == "linux") and not swgl and not fission and not webrender: ["CRASH", "ERROR"]
+    if debug and (os == "linux") and not swgl and fission: ["CRASH", "TIMEOUT", "ERROR"]
+    if not debug and (os == "linux") and not webrender: ["OK", "TIMEOUT"]
     if not debug and (os == "win") and (processor == "x86"): ["TIMEOUT", "OK"]
-    if not debug and (os == "linux") and not webrender: ["TIMEOUT", "OK"]
-    if not debug and (os == "linux") and webrender: TIMEOUT
-    if debug and (os == "linux"): TIMEOUT
-    if not debug and (os == "android"): OK
-    CRASH
+    if debug and (os == "android") and webrender: ["CRASH", "OK"]
+    if debug and (os == "android") and not webrender: CRASH
+    if debug and (os == "linux") and swgl: ["CRASH", "OK", "TIMEOUT"]
+    if debug and (os == "win"): CRASH
+    if debug and (os == "mac"): CRASH
 
 [basic-auth.any.sharedworker.html?wss]
 
@@ -36,16 +41,21 @@
 
 [basic-auth.any.worker.html?wpt_flags=h2]
   expected:
-    if not debug and (os == "linux") and webrender: TIMEOUT
-    if not debug and (os == "linux") and not webrender: ["TIMEOUT", "OK"]
-    if not debug and (os == "win") and (processor == "x86"): ["TIMEOUT", "OK"]
+    if debug and (os == "linux") and webrender and not swgl and fission: ["CRASH", "TIMEOUT"]
+    if debug and (os == "linux") and webrender and not swgl and not fission: ["CRASH", "TIMEOUT"]
+    if debug and (os == "linux") and not webrender: ["CRASH", "ERROR"]
     if not debug and (os == "win") and (processor == "x86_64"): OK
-    if debug and (os == "linux"): TIMEOUT
+    if not debug and (os == "win") and (processor == "x86"): ["OK", "TIMEOUT"]
+    if not debug and (os == "linux") and not fission: ["OK", "TIMEOUT"]
+    if not debug and (os == "linux") and fission: OK
     if not debug and (os == "android"): OK
+    if not debug and (os == "mac"): OK
     CRASH
   [HTTP basic authentication should work with WebSockets]
     expected:
-      if os == "linux": ["PASS", "FAIL"]
+      if (os == "linux") and webrender and not fission: ["PASS", "FAIL", "TIMEOUT"]
+      if (os == "linux") and webrender and fission: ["PASS", "FAIL"]
+      if (os == "linux") and not webrender: ["PASS", "FAIL"]
 
 
 [basic-auth.any.html?wss]
@@ -54,14 +64,20 @@
 
 [basic-auth.any.serviceworker.html?wpt_flags=h2]
   expected:
-    if not debug and (os == "linux") and not webrender: ["TIMEOUT", "OK"]
-    if not debug and (os == "linux") and webrender: TIMEOUT
-    if not debug and (os == "win") and (processor == "x86"): ["TIMEOUT", "OK"]
+    if debug and (os == "linux") and webrender and not fission and not swgl: ["CRASH", "OK", "ERROR", "TIMEOUT"]
+    if debug and (os == "linux") and webrender and fission: ["CRASH", "ERROR", "TIMEOUT"]
+    if debug and (os == "linux") and not webrender: ["CRASH", "TIMEOUT"]
+    if not debug and (os == "win") and (processor == "x86"): ["OK", "TIMEOUT"]
     if not debug and (os == "win") and (processor == "x86_64"): OK
+    if not debug and (os == "linux") and not fission: ["OK", "TIMEOUT"]
+    if not debug and (os == "linux") and fission: OK
     if not debug and (os == "android"): OK
-    if debug and (os == "linux"): TIMEOUT
+    if not debug and (os == "mac"): OK
     CRASH
   [HTTP basic authentication should work with WebSockets]
     expected:
-      if os == "linux": ["PASS", "FAIL"]
+      if (os == "linux") and not debug and webrender and not fission: ["PASS", "FAIL", "TIMEOUT"]
+      if (os == "linux") and not debug and webrender and fission: ["PASS", "FAIL"]
+      if (os == "linux") and not debug and not webrender: ["PASS", "FAIL"]
+      if (os == "linux") and debug: ["FAIL", "PASS"]
 